synapse/config/server.py

```diff
@@ -111,13 +111,15 @@ def read_config(self, config):
         self.admin_contact = config.get("admin_contact", None)
 
         # FIXME: federation_domain_whitelist needs sytests
-        self.federation_domain_whitelist = None
```

Comment:
why is this code being changed?

Reply:
Left over from previous refactor that was reverted. Have changed code back so there's less diff now.